templates/components/itilobject/timeline/form_validation.html.twig

```diff
@@ -181,18 +181,20 @@
                 <h3 class="cart-title">{{ __("Validation request") }}</h3>
             </div>
             <div class="card-body">
-               {{ fields.dropdownField(
-                  'ITILValidationTemplate',
-                  'itilvalidationtemplates_id',
-                  '',
-                  _n('Template', 'Templates', 1),
-                  {
-                     'full_width': true,
-                     'on_change': 'itilvalidationtemplate_update' ~ rand ~ '(this.value);',
-                     'entity': item.fields['entities_id'],
-                     'rand': rand,
-                  },
-                ) }}
+                {% if item.isNewItem() %}
```

Comment:
Why limit the usage of template to just created tickets ?

Reply:
I guess, it should be ```{% if subitem.isNewItem() %}```
In this case, the template choice is available only where creating an approval (not a ticket).

Am I right ?

Reply:
Indeed, it seems that the condition should be checked on `subitem`.